### ISSUE TITLE
Verifying the SigningCertURL

### ISSUE THREAD

**Issue by jplindstrom** [rizen/AWS-SNS-Verify#2]
Hi,

I'd like to add validation of the SigningCertURL, so that it's a valid AWS SNS url and not a random client provided URL.

Although it's a bit naff as a security measure, this is done in the (not sure if official) AWS PHP client here: https://github.com/aws/aws-php-sns-message-validator/blob/master/src/MessageValidator.php#L22

```
     * @var string  A pattern that will match all regional SNS endpoints, e.g.:
     *                  - sns.<region>.amazonaws.com        (AWS)
     *                  - sns.us-gov-west-1.amazonaws.com   (AWS GovCloud)
     *                  - sns.cn-north-1.amazonaws.com.cn   (AWS China)
     */
    private static $defaultHostPattern
        = '/^sns\.[a-zA-Z0-9\-]{3,}\.amazonaws\.com(\.cn)?$/';

```

Are you ok with this, so I can go ahead with it?

**Comment by rizen** [comment]
As long as you provide a test to go with it, I'm cool. Go for it.

**Comment by jplindstrom** [comment]
Hey @rizen, would you mind having a look at the pull request #3 that implements this?

**Comment by rizen** [comment]
Sorry for the delay. Completely forgot about this. It has been released to cpan as 0.0102.

**Comment by jplindstrom** [comment]
Very cool, thanks!
